vncviewer/cocoa.mm

```diff
@@ -145,9 +145,9 @@ int cocoa_is_keyboard_event(const void *event)
   nsevent = (NSEvent*)event;
 
   switch ([nsevent type]) {
-  case NSKeyDown:
-  case NSKeyUp:
-  case NSFlagsChanged:
+  case NSEventTypeKeyDown:
+  case NSEventTypeKeyUp:
+  case NSEventTypeFlagsChanged:
```

Comment:
What do these changes mean in terms of minimum macOS SDK version?

Reply:
macOS 10.12+

Reply:
and currently our minimum requirements are macOS 10.6+ @CendioOssman?

Reply:
Yeah, that's a bit too new for us for now. Has Apple actually removed the old ones? Or is it just a warning?

Reply:
Warning promoted to error by default clang11 behavior. 10.12 is a 2016 macOS release; what is the line for too new?

Reply:
Right now we're making sure it compiles with the 10.6 SDK, and runs on 10.9+.

If clang errors on those I think we'll have to use `#ifdef`s.